lib/report_plugins/daytoday.js

```diff
@@ -423,6 +449,7 @@ daytoday.report = function report_daytoday(datastorage,sorteddaystoshow,options)
       });
     }
     
+    
     data.treatments.forEach(function (treatment) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (12)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2307126404/issues/source?bid=2452379&fileBranchId=3191689#l453)